.github/workflows/linux.yml

```diff
@@ -0,0 +1,84 @@
+---
+name: Linux CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+jobs:
+  tests:
+    name: "${{ matrix.env.FLAVOR }} ${{ matrix.env.BOOST }} ${{ matrix.env.COMPILER }}"
+    runs-on: "ubuntu-latest"
+
+    container:
+      image: bondciimages.azurecr.io/ubuntu-1604:build-32869346
+
+    strategy:
+      fail-fast: false
+      matrix:
+        env:
+          - { FLAVOR: cs }
+          - { FLAVOR: cpp-core, BOOST: 1.66.0, COMPILER: clang }
+          - { FLAVOR: cpp-grpc, BOOST: 1.66.0, COMPILER: clang }
+          - { FLAVOR: hs }
+          - { FLAVOR: java }
+
+          - { FLAVOR: cpp-core, BOOST: 1.65.1, COMPILER: clang }
```

Comment:
The variants covered by the cron workflow can be removed from the CI workflow.

Reply:
done